.jazzy.yml

```diff
@@ -0,0 +1,181 @@
+author: Nataly Carbonell, Patrick Freed, Kaitlin Mahar
```

Comment:
this file is mostly a copy paste from the driver one, with the addition of the "Vapor Extensions" category